test/mocha/25-http-services.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2017-2018 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const async = require('async');
+const bedrock = require('bedrock');
+const brIdentity = require('bedrock-identity');
+// const brLedger = require('bedrock-ledger-node');
+const brLedgerAgent = require('bedrock-ledger-agent');
+// const config = bedrock.config;
+const {express} = require('bedrock-express');
+const helpers = require('./helpers');
+const jsigs = require('jsonld-signatures');
+const mockData = require('./mock.data');
+// const querystring = require('querystring');
+let request = require('request');
+request = request.defaults({json: true, strictSSL: false});
+// require('request-debug')(request);
+// const url = require('url');
+// const uuid = require('uuid/v4');
+
+jsigs.use('jsonld', bedrock.jsonld);
+
+// const urlObj = {
+//   protocol: 'https',
+//   host: config.server.host,
+//   pathname: config['ledger-agent'].routes.agents
+// };
+
+// register a mock service
+const api = {router: express.Router()};
+api.router.get('/', (req, res) => {
+  // `ledgerAgent` is added to the `req` object
+  should.exist(req.ledgerAgent);
+  res.json({success: true});
+});
+brLedgerAgent.use('foo', {api, type: 'service'});
```

Comment:
I think `service` is perhaps too generic a name here given that the "services" will all be mounted on the ledger-agent/plugins/* endpoint. The type may need to be more specific to ledger agent. Others may disagree.

cc: @msporny, @davidlehn